CDCgov/prime-reportstream
# Onboarding ops/6221/wy not otc to feed/jorge

This PR edits the settings for WY by filtering out OTC data from their secondary feed.

Test Steps:
1. Generate fake data for WY
2. Edit `test_authorized_for_(home/otc/unproctored)` by using code Y
3. Run docker
4. Run reloadsettings
5. send file to api
6. check that the secondary feed filtered out the file

## Changes
Added the following filters under the routing filter:
 - "matches(test_authorized_for_home, N)"
 - "matches(test_authorized_for_otc, N)"
 - "matches(test_authorized_for_unproctored, N)"

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #6493
